integration_tests/src/main/python/json_test.py

```diff
@@ -200,6 +202,49 @@ def test_json_ts_formats_round_trip(spark_tmp_path, date_format, ts_part, v1_ena
                     .json(data_path),
             conf=updated_conf)
 
+@allow_non_gpu('BatchScanExec', 'FileSourceScanExec', 'ProjectExec')
```

Comment:
I'm not a fan of adding more to the ignore list than is necessary, and I think this might be a bit more readable (and easier to run a specific test) if this were refactored into two tests, a v1 test and a v2 test, with appropriate non-GPU allows for each.  The code can remain common, passing the class names as args to the refactored, common code.